### PR TITLE
cleanup for howtos3

### PR DESCRIPTION
**Pull request by jesgreco**
cleaning up drag_and_drop_deprecated.rst doc

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jesgreco!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@openedx/docs-openedx-org-maintainers`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

**Comment by sarina** [comment]
Let's stop committing to this branch as it's getting big.

**Comment by sarina** [comment]
Let's stop committing to this branch as it's getting big. Focus on fixing the build errors, please!

**Comment by jesgreco** [comment]
@sarina errors were fixed, you can proceed to merge this PR. Thanks!

File: source/educators/how-tos/data/manage_course_grades.rst
Comment (sarina) [inline source/educators/how-tos/data/manage_course_grades.rst:73]:
```suggestion
   Best practice is that you announce any unavoidable grading related changes to
```

File: source/educators/how-tos/data/manage_course_grades.rst
Comment (sarina) [inline source/educators/how-tos/data/manage_course_grades.rst:84]:
```suggestion
results for all learners in the course, including unenrolled learners. Best practice is to generate```

File: source/educators/how-tos/data/manage_course_grades.rst
Comment (sarina) [inline source/educators/how-tos/data/manage_course_grades.rst:89]:
```suggestion
your instance's servers. The complexity of your grading configuration and the number of
```

File: source/educators/how-tos/data/manage_course_grades.rst
Comment (sarina) [inline source/educators/how-tos/data/manage_course_grades.rst:254]: I think we can remove everything about master's courses as that's edx.org-specific, but I'd check in with @jswope00 about that.

File: source/educators/how-tos/data/manage_course_grades.rst
Comment (sarina) [inline source/educators/how-tos/data/manage_course_grades.rst:269]: Same here. I think Master's only is edx.org-specific. @jswope00 ?

File: source/educators/how-tos/data/manage_course_grades.rst
Comment (sarina) [inline source/educators/how-tos/data/manage_course_grades.rst:289]:
```suggestion
   * **username**: The user's username
```

File: source/educators/how-tos/data/manage_course_grades.rst
Comment (sarina) [inline source/educators/how-tos/data/manage_course_grades.rst:343]:
```suggestion
* user ID
```

File: source/educators/how-tos/data/manage_course_grades.rst
Comment (sarina) [inline source/educators/how-tos/data/manage_course_grades.rst:345]:
```suggestion
* username
```

File: source/educators/how-tos/data/manage_course_grades.rst
Comment (sarina) [inline source/educators/how-tos/data/manage_course_grades.rst:455]:
```suggestion
*This feature is supported in courses published after June 1, 2019. To enable for an older course, first Publish the course from Studio.*
```

File: source/educators/how-tos/data/manage_course_grades.rst
Comment (sarina) [inline source/educators/how-tos/data/manage_course_grades.rst:891]:
```suggestion
When you create each of the problems for a course, the platform assigns a unique
```

File: source/educators/how-tos/drag_and_drop_deprecated.rst
Comment (sarina) [inline source/educators/how-tos/drag_and_drop_deprecated.rst:43]: I think we can just remove this page, since the problem type has been deprecated for quite some time.

File: source/educators/how-tos/update_rerun_course.rst
Comment (sarina) [inline source/educators/how-tos/update_rerun_course.rst:90]:
```suggestion
```
Just remove this line

File: source/educators/how-tos/update_rerun_course.rst
Comment (sarina) [inline source/educators/how-tos/update_rerun_course.rst:92]:
```suggestion
    provide access to the *Open edX Learner's Guide*. 
```

File: source/educators/how-tos/update_rerun_course.rst
Comment (sarina) [inline source/educators/how-tos/update_rerun_course.rst:93]:
```suggestion

```

**Review by sarina — CHANGES_REQUESTED** [review]

File: source/educators/how-tos/communication/administer_discussions.rst
Comment (jesgreco) [reply source/educators/how-tos/communication/administer_discussions.rst]: Done

File: source/educators/how-tos/course_development/exercise_tools/add_numerical_input_problem.rst
Comment (jesgreco) [reply source/educators/how-tos/course_development/exercise_tools/add_numerical_input_problem.rst]: removed

File: source/educators/how-tos/data/course_answers.rst
Comment (jesgreco) [reply source/educators/how-tos/data/course_answers.rst]: done

File: source/educators/how-tos/communication/administer_discussions.rst
Comment (sarina) [reply source/educators/how-tos/communication/administer_discussions.rst:11]: @jesgreco - when I make a note like this can you please search the whole file? There are two more that I see.

File: source/educators/how-tos/data/manage_course_grades.rst
Comment (sarina) [reply source/educators/how-tos/data/manage_course_grades.rst:252]: @jesgreco please remove all content about Master's courses

File: source/educators/how-tos/update_rerun_course.rst
Comment (sarina) [inline source/educators/how-tos/update_rerun_course.rst:88]: Actually, delete everything under `.. only:: Partners`, please.

Can you please check the other files in this pull request to see if they have anything specified as `.. only:: Partners`?

Comment (jesgreco) [reply source/educators/how-tos/update_rerun_course.rst:88]: done! I've reviewed the 27 files and no further reference for that.

File: source/educators/navigation/components_activities.rst
Comment (sarina) [inline source/educators/navigation/components_activities.rst:99]: I think you want to keep this in, and delete L104 (the file you deleted)

Comment (jesgreco) [reply source/educators/navigation/components_activities.rst:99]: Yes, but still the error is happening.

**Comment by jesgreco** [comment]
@sarina you can proceed to the merge.

**Review by sarina — APPROVED** [review]
My comments are addressed, but I asked @jswope00 to review this PR as well. John can merge once his review is completed. (John, I suggest you "squash and merge" so we don't have a really long commit history)

File: source/educators/how-tos/data/manage_course_grades.rst
Comment (jswope00) [reply source/educators/how-tos/data/manage_course_grades.rst:252]: agree that references to "Master's courses" should be removed.

File: source/educators/how-tos/update_rerun_course.rst
Comment (jswope00) [reply source/educators/how-tos/update_rerun_course.rst:88]: I think the ".. only::" comment still needs to be deleted, yes?

**Review by jswope00 — CHANGES_REQUESTED** [review]

File: source/educators/how-tos/update_rerun_course.rst
Comment (jesgreco) [reply source/educators/how-tos/update_rerun_course.rst]: Done

**Comment by jesgreco** [comment]
@jswope00 I've just finished with your comments, you can proceed to merge the PR.

**Comment by sarina** [comment]
@jesgreco please do not add more stuff to a branch after the review has happened!

**Comment by jesgreco** [comment]
Hi, @sarina I only work on the stuff asked by John. Sorry, but there were some conflicts also that needed to be solved.

**Comment by sarina** [comment]
Ah, it looked like you were pushing unrelated commits. Thank you.

**Review by jswope00 — APPROVED** [review]

**Pull request merged**
